selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AssistantFindPanel.java

```diff
@@ -139,6 +139,18 @@ public void clickOnActionNodeWithText(String visibleText) {
     throw new RuntimeException(exceptionMessage);
   }
 
+  public void clickOnActionNodeWithTextEqualsTo(String visibleText) {
```

Comment:
To avoid confusion with `clickOnActionNodeWithText()` method I would propose to rename:
`clickOnActionNodeWithText` > `clickOnActionNodeWithTextContains`